src/desktop/components/main_layout/stylesheets/buttons.styl

```diff
@@ -81,36 +81,35 @@
   margin 0 5px
 
 .avant-garde-button
+  text('mediumText')
```

Comment:
Peril is about to merge a change to this stanza as well, so head's up that one of us might have merge conflicts.

Reply:
👍